### ISSUE TITLE
Ftp-sync: Uploading index.php failed: Error: Please tell me who you are

### ISSUE THREAD

**Issue by ghost** [lukasz-wronski/vscode-ftp-sync#341]
I get this message when I try to upload on save. Brand new install of vscode, this is literally the first thing I tried to do.
Please let me know what you need.

**Ftp-sync: Uploading index.php failed: Error: Please tell me who you are**

**Comment by ToniGuga** [comment]
I'm having the same problem... Any idea? 
iI'm using latest vscode with latest ftp sync versione on windows. My colleague on mac is not having this problem.

**Comment by gmagdics** [comment]
Same issue, on a mac, after having worked fine for a couple of days without problems with a freshly installed vscode. Didn't change anything, then this came. Message disappeared after restarting vscode, but upload kept hanging. Reinstalling the extension solved it. Not sure for how long though.

**Comment by HeadStudios** [comment]
Yep, same issue here. Real shame - I'll check my config because I know files were getting uploaded when I was doing it manually - then I started doing it automatically and it stopped working... also interesting behavior... when I auto save it doesn't work but when I manually upload it does work.. very weird!
